response_operations_ui/forms.py

```diff
@@ -82,6 +89,31 @@ def __init__(self, form, default_values=None):
             self.email.data = default_values.get('emailAddress')
             self.telephone.data = default_values.get('telephone')
 
+    @staticmethod
+    def validate_first_name(form, field):
+        first_name = field.data
+        if first_name is None:
+            raise ValidationError('Enter a fist name')
+
+    @staticmethod
+    def validate_last_name(form, field):
+        last_name = field.data
+        if last_name is None:
+            raise ValidationError('Enter a last name')
+        if len(last_name) >= 254:
+            raise ValidationError("Last name must be fewer than 254 characters")
+
+    @staticmethod
+    def validate_email_address(form, field):
```

Comment:
This needs to be called `validate_email` to work properly - the way WTForms works is that any method called `validate_{field name}` is called on submit.